integrations/langfuse/tests/test_tracer.py

```diff
@@ -367,7 +373,8 @@ def test_update_span_flush_disable(self, monkeypatch):
         monkeypatch.setenv("HAYSTACK_LANGFUSE_ENFORCE_FLUSH", "false")
         tracer_mock = Mock()
 
-        from haystack_integrations.tracing.langfuse.tracer import LangfuseTracer
+        from haystack_integrations.tracing.langfuse.tracer import \
+            LangfuseTracer
```

Comment:
Small nit, why add the line break?

Reply:
Hm, must have been black or ruff, let me see